src/Fixer/Basic/BracesFixer.php

```diff
@@ -846,6 +846,11 @@ private function ensureWhitespaceAtIndexAndIndentMultilineComment(Tokens $tokens
 
         $nextToken = $tokens[$nextTokenIndex];
         if ($nextToken->isComment()) {
+            $previousToken = $tokens[$nextTokenIndex - 1];
+            // do not indent comments starting at the beginning of line - non-indented comments should not change indentation
+            if ($previousToken->isWhitespace() && 1 === preg_match('/\R$/', $previousToken->getContent())) {
+                return;
```

Comment:
Actually I'm not sure what to do here. Absence of indentation in cases like @keradus's example should be fixed IMO, but there is no way to know if the comment is a real comment or just "removed" code. Though I think this case is pretty rare and unindented comments will most of the time be "removed" code.

On the other hand, when using a version control system like Git, one could say that commenting outcode instead of completely removing it is a bad practice. So maybe adding a new option here would be better?